### PR TITLE
CLI V2

### PR DESCRIPTION
**Pull request by ianjennings**
[Able to pipe a replay decription!](https://www.loom.com/share/b7236da2aa6d418488d281b86413a7c4)

  Watch [Able to pipe a replay decription!](https://www.loom.com/share/b7236da2aa6d418488d281b86413a7c4) on Replayable, umm wait, on Loom
  
  Depends on https://github.com/replayableio/desktop/pull/411

The readme will explain most of what's going on here:
https://github.com/replayableio/cli/blob/e752d9e1900382ff642c574514eddf1608bc477d/README.md

### CODE REVIEW

**Comment by ianjennings** [comment]
Draft because I need to move from TCP to Websockets

File: lib.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        console.log(
+          "Could not connect to Replayable Desktop App. Is it running?"
+        );
+        console.log(
+          "You may need to download and install the app from https://bit.ly/3erITXJ"
+        );
+      }
+    });
+    ipc.of.replayable.on("disconnect", function () {
+      console.log("Disconnected from replayable".notice);
```
Comment (AyoubElk) [inline lib.js]: `.notice` looks like a typo?

File: README.md [hunk 2]
```diff
@@ -1,51 +1,129 @@
...
-## Append a 30 second replay to a commit
+### Attach the last 100 CLI commands to the replay description
 
 ```sh
-git commit -am "`replayable`"
+$ history -100 | replayable
 ```
 
-## Demo
+### Embed a logfile in the replay description
```
Comment (AyoubElk) [inline README.md]: Missing example command.

File: lib.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  return new Promise((resolve, reject) => {
+    ipc.connectTo("replayable");
+    ipc.of.replayable.on("connect", resolve);
+    ipc.of.replayable.on("error", (e) => {
+      if (e.code === "ENOENT") {
+        console.log(
+          "Could not connect to Replayable Desktop App. Is it running?"
+        );
+        console.log(
+          "You may need to download and install the app from https://bit.ly/3erITXJ"
```
Comment (AyoubElk) [inline lib.js]: Link doesn't work
image

**Review by AyoubElk — CHANGES_REQUESTED** [review]

File: lib.js (hunk 3 above)
Comment (ianjennings) [reply lib.js]: Great find.

**Pull request merged**